plugins/analytics/public/onyphe.py

```diff
@@ -0,0 +1,369 @@
+import sys
+import json
+import logging
+import requests
+from requests.utils import quote
+
+from core.analytics import OneShotAnalytics
+from core.errors import ObservableValidationError
+from core.observables import Hostname, Ip
+
+default_types = {
+    "Hostname": Hostname,
+    "Ip": Ip,
+}
+
+if sys.version_info.major == 2:
```

Comment:
Please use six for this: https://pythonhosted.org/six/

Reply:
ok done,  was to not have six as deps in original commit :)